background.ts

```diff
@@ -1493,25 +1493,28 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
 
   async listExtensions() {
     const extensionManager = await getExtensionManager();
-    const extensions = await extensionManager?.getInstalledExtensions() ?? [];
+
+    if (!extensionManager) {
+      return undefined;
+    }
+    const extensions = await extensionManager.getInstalledExtensions();
     const entries = await Promise.all(extensions.map(async x => [x.id, {
       version:  x.version,
       metadata: await x.metadata,
       labels:   await x.labels,
     }] as const));
 
-    return Promise.resolve(Object.fromEntries(entries));
+    return Object.fromEntries(entries);
   }
 
   async installExtension(image: string, state: 'install' | 'uninstall'): Promise<{status: number, data?: any}> {
     const em = await getExtensionManager();
```

Comment:
Nit-picking, and I know this is pre-existing code, but `em` is being called `extensionManager` in the function just above. Maybe standardize on one name?

Reply:
Eh, I'm just going to skip this for now.